moment_kinetics/src/input_structs.jl

```diff
@@ -153,8 +155,9 @@ end
 
 """
 """
-@enum electron_physics_type boltzmann_electron_response boltzmann_electron_response_with_simple_sheath
-export electron_physics_type, boltzmann_electron_response, boltzmann_electron_response_with_simple_sheath
+const boltzmann_electron_response = "boltzmann_electron_response"
```

Comment:
@johnomotani @LucasMontoya4 @mabarnes 

FYI: The changes on these lines are all we need to make (+ a type change in species_composition) to avoid using `enum` types. Personally, I prefer this option, as it is much more transparent (in my opinion). I am not used to Julia's ability to convert strings to symbols and the reverse. Besides avoiding troubles with I/O (#236), I think this way of determining the electron_physics variable is a better way of self-documenting the code so that it is grep-able.

Reply:
I have follow @johnomotani and reinstated the enums.